package.json

```diff
@@ -64,6 +64,7 @@
     ]
   },
   "dependencies": {
+    "babel-jest": "^24.9.0",
```

Comment:
is there a reason you adding babel-jest especially since we are not using babel???

Reply:
This was actually added by you. You can pull from development and hover on the line. But I will remove it nevertheless